pkg/configuration/configuration.go

```diff
@@ -0,0 +1,63 @@
+package configuration
+
+import (
+	"fmt"
+	"os"
+)
+
+type ErrorCollector struct {
+	Missing []string
+}
+
+func (e *ErrorCollector) Register(name string) {
+	e.Missing = append(e.Missing, name)
+}
+
+func (e *ErrorCollector) Error() error {
+	if len(e.Missing) == 0 {
+		return nil
+	} else {
+		var message string
+		for _,name := range e.Missing {
+			message += fmt.Sprintf("env variable with name %s was not found, ", name)
+		}
+		return fmt.Errorf(message)
+	}
```

Comment:
`else` clause can be omitted as `if` section always returns.

```suggestion
	}
	var message string
	for _,name := range e.Missing {
		message += fmt.Sprintf("env variable with name %s was not found, ", name)
	}
	return fmt.Errorf(message)
```